### PR TITLE
feat(route): 添加一个lovelive官网NEWS信息获取的路由

### PR DESCRIPTION
**Pull request by axojhf**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/lovelive-anime/news
/lovelive-anime/news/detail
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [x] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: docs/anime.md [hunk 1]
```diff
@@ -347,6 +347,12 @@ Sources
 
 </Route>
 
+## lovelive-anime
+
+### Love Live 官网最新News
+
+<Route author="axojhf" example="/lovelive-anime/news" path="/lovelive-anime/news/:option?" :paramsDesc="['option为detail时抓取全文']" />
```
Comment (TonyRL) [inline docs/anime.md:354]:
```suggestion
<Route author="axojhf" example="/lovelive-anime/news" path="/lovelive-anime/news/:option?" :paramsDesc="['`option` 为 `detail` 时抓取全文']" />
```

File: docs/anime.md [hunk 2]
```diff
@@ -347,6 +347,12 @@ Sources
 
 </Route>
 
+## lovelive-anime
```
Comment (TonyRL) [inline docs/anime.md:350]: Please add en docs as well.

File: lib/v2/lovelive-anime/news.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+const { art } = require('@/utils/render');
+const renderDescription = (desc) => art(path.join(__dirname, 'templates/description.art'), desc);
+
+module.exports = async (ctx) => {
+    const rootUrl = 'https://www.lovelive-anime.jp/news/';
+
+    const response = await got({
+        method: 'get',
+        url: rootUrl,
+    });
```
Comment (TonyRL) [inline lib/v2/lovelive-anime/news.js:13]:
```suggestion
    const response = await got(rootUrl);
```

File: lib/v2/lovelive-anime/news.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    let items = pageFace;
+
+    if (ctx.params.option === 'detail') {
+        items = await Promise.all(
+            pageFace.map((item) =>
+                ctx.cache.tryGet(item.link, async () => {
+                    const detailResp = await got({
+                        method: 'get',
+                        url: item.link,
+                    });
```
Comment (TonyRL) [inline lib/v2/lovelive-anime/news.js:43]:
```suggestion
                    const detailResp = await got(item.link);
```

File: lib/v2/lovelive-anime/news.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+                    const $ = cheerio.load(detailResp.data);
+
+                    const content = $('div.p-page__detail.p-article');
+                    for (const v of content.find('img')) {
+                        v.attribs.src = 'https://www.lovelive-anime.jp' + v.attribs.src;
+                    }
+                    // item.description = content.html()
+                    item.description = renderDescription({
+                        content: content.html(),
+                    });
```
Comment (TonyRL) [inline lib/v2/lovelive-anime/news.js:53]:
```suggestion
                    item.description = content.html();
```
No need to use `art` when rendering raw HTML from target site.

File: lib/v2/lovelive-anime/templates/description.art [hunk 6]
```diff
@@ -0,0 +1,5 @@
+{{if content}}
+  {{@ content}}
+{{else}}
+{{title}}<br><img src="{{imglink}}">
+{{/if}}
```
Comment (TonyRL) [inline lib/v2/lovelive-anime/templates/description.art:5]:
```suggestion
{{title}}<br><img src="{{imglink}}">
```

File: docs/anime.md (hunk 2 above)
Comment (axojhf) [reply docs/anime.md:350]: > Please add en docs as well.

OK

**Pull request merged**